gpu/containers/include/pcl/gpu/containers/impl/device_array.hpp

```diff
@@ -34,92 +34,258 @@
  *  Author: Anatoly Baskeheev, Itseez Ltd, (myname.mysurname@mycompany.com)
  */
 
-
 #ifndef PCL_GPU_CONTAINER_DEVICE_ARRAY_IMPL_HPP_
 #define PCL_GPU_CONTAINER_DEVICE_ARRAY_IMPL_HPP_
 
+namespace pcl {
 
-namespace pcl
-{
+namespace gpu {
 
-namespace gpu
-{
+/////////////////////  Inline implementations of DeviceArray
+///////////////////////////////////////////////
 
-/////////////////////  Inline implementations of DeviceArray ////////////////////////////////////////////
-
-template<class T> inline DeviceArray<T>::DeviceArray() {}
-template<class T> inline DeviceArray<T>::DeviceArray(std::size_t size) : DeviceMemory(size * elem_size) {}
-template<class T> inline DeviceArray<T>::DeviceArray(T *ptr, std::size_t size) : DeviceMemory(ptr, size * elem_size) {}
-template<class T> inline DeviceArray<T>::DeviceArray(const DeviceArray& other) : DeviceMemory(other) {}
-template<class T> inline DeviceArray<T>& DeviceArray<T>::operator=(const DeviceArray& other)
-{ DeviceMemory::operator=(other); return *this; }
+template <class T>
+inline DeviceArray<T>::DeviceArray()
+{}
```

Comment:
Let's separate these functions with newlines

Reply:
Yes, that is an excellent idea! I wonder why clang-format cannot do that automatically for us, but I couldn't find a corresponding setting. You don't know one either?

Reply:
I don't think there's an option for that: https://releases.llvm.org/11.0.0/tools/clang/docs/ClangFormatStyleOptions.html